<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 6.1 (Python 3.11.4 on darwin)" generated="20230718 21:34:09.394" rpa="false" schemaversion="4">
<suite id="s1" name="LoginTest" source="/Users/ayushisingh/Documents/RobotFrameworkwithPython/TestCases/LoginTest.robot">
<test id="s1-t1" name="LoginTest" line="11">
<kw name="Open My Browser ${URL}">
<arg>${Browser}</arg>
<msg timestamp="20230718 21:34:09.687" level="FAIL">No keyword with name 'Open My Browser ${URL}' found.</msg>
<status status="FAIL" starttime="20230718 21:34:09.687" endtime="20230718 21:34:09.687"/>
</kw>
<kw name="Enter UserName">
<arg>${UserName}</arg>
<status status="NOT RUN" starttime="20230718 21:34:09.689" endtime="20230718 21:34:09.689"/>
</kw>
<status status="FAIL" starttime="20230718 21:34:09.684" endtime="20230718 21:34:09.689">No keyword with name 'Open My Browser ${URL}' found.</status>
</test>
<status status="FAIL" starttime="20230718 21:34:09.395" endtime="20230718 21:34:09.693"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="LoginTest">LoginTest</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20230718 21:34:09.456" level="ERROR">Error in file '/Users/ayushisingh/Documents/RobotFrameworkwithPython/TestCases/LoginTest.robot' on line 3: Resource file 'PageResources/LoginPageKeywords.robot' does not exist.</msg>
</errors>
</robot>
